.jenkins/dsl/jobs.groovy

```diff
@@ -18,6 +18,27 @@ def getDefaultJobParams(String repoName = 'optaplanner') {
             repository: repoName,
             credentials: "${GIT_AUTHOR_CREDENTIALS_ID}",
             token_credentials: "${GIT_AUTHOR_TOKEN_CREDENTIALS_ID}"
+        ],
+        env: [:],
+        pr: [:]
+    ]
+}
+
+Map getMultijobPRConfig() {
```

Comment:
How does this MultiJob pipeline relate to the Build Chain Tool based on GitHub Actions? Is there an agreement to replace the Build Chain tool with the multijob pipeline?

Reply:
This is Kogito project and its PR checks.
Problem that was arising is having a downstream project and then you need to restart the whole build&tests.
The current BuildChain has 2 drawbacks: we have limited number of nodes available on GHAction and are seeing some jobs needed to wait for a build to run + if a downstream project is failing, you need to restart everything.
This is an alternative to the BuildChain and the full Jenkins PR check (I tend to say that it should replace this latter one in Kogito project). 
This is for now tighten to the Kogito project and is not meant yet to replace the BuildChain.

And with discussion it came also to the point that having 2 different infrastructures: BuildChain on GHAction and Jenkins is not a bad thing as if any of those is failing, then we have a backup ... since the latest Jenkins infra problems, we decided to setup the BuildChain for Kogito project as a backup)